nbs/examples/train_imagenette.py

```diff
@@ -59,7 +68,7 @@ def main(
     elif opt=='sgd'   : opt_func = partial(SGD, mom=mom)
     elif opt=='ranger': opt_func = partial(ranger, mom=mom, sqr_mom=sqrmom, eps=eps, beta=beta)
 
-    dls = rank0_first(get_dls, size, woof, bs, sh=sh, workers=workers)
+    dls = rank0_first(get_dls, size, woof, pct_noise, bs, sh=sh, workers=workers)
     pr(f'epochs: {epochs}; lr: {lr}; size: {size}; sqrmom: {sqrmom}; mom: {mom}; eps: {eps}')
```

Comment:
```suggestion
    pr(f'pct_noise: {pct_noise}; epochs: {epochs}; lr: {lr}; size: {size}; sqrmom: {sqrmom}; mom: {mom}; eps: {eps}')
```
Printing out the amount of noise used at the beginning of the run